### PR TITLE
Issue 69

### PR DESCRIPTION
**Pull request by mykolalosev**
### Description of your changes

Add the following resources:
Moving 1 resource in the `batch` group:
- [x] [`aws_batch_scheduling_policy`](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/batch_scheduling_policy)

Moving 2 resources in the `budgets` group:
- [x] [`aws_budgets_budget`](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/budgets_budget)
- [x] [`aws_budgets_budget_action`](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/budgets_budget_action)

Moving 7 resources in the `chime` group:
- [x] [`aws_chime_voice_connector`](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/chime_voice_connector)
- [x] [`aws_chime_voice_connector_group`](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/chime_voice_connector_group)
- [x] [`aws_chime_voice_connector_logging`](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/chime_voice_connector_logging)
- [x] [`aws_chime_voice_connector_origination`](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/chime_voice_connector_origination)
- [x] [`aws_chime_voice_connector_streaming`](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/chime_voice_connector_streaming)
- [x] [`aws_chime_voice_connector_termination`](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/chime_voice_connector_termination)
- [x] [`aws_chime_voice_connector_termination_credentials`](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/chime_voice_connector_termination_credentials)

Moving 2 resources in the `cloud9` group:
- [x] [`aws_cloud9_environment_ec2`](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/cloud9_environment_ec2)
- [x] [`aws_cloud9_environment_membership`](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/cloud9_environment_membership)

Moving 1 resource in the `cloudcontrol` group:
- [x] [`aws_cloudcontrolapi_resource`](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/cloudcontrolapi_resource)

Fixes https://github.com/upbound/provider-aws/issues/69

I have:

- [x] Run `make reviewable test` to ensure this PR is ready for review.

### How has this code been tested
Manually and Uptest:
aws_batch_scheduling_policy: https://github.com/upbound/provider-aws/actions/runs/3563209290
aws_budgets_budget: https://github.com/upbound/provider-aws/actions/runs/3563300621
aws_budgets_budget_action: https://github.com/upbound/provider-aws/actions/runs/3564412062
aws_chime_voice_connector: https://github.com/upbound/provider-aws/actions/runs/3583132957
aws_chime_voice_connector_group: https://github.com/upbound/provider-aws/actions/runs/3583140400
aws_chime_voice_connector_logging: https://github.com/upbound/provider-aws/actions/runs/3583148750
aws_chime_voice_connector_origination: https://github.com/upbound/provider-aws/actions/runs/3566280289
aws_chime_voice_connector_streaming: https://github.com/upbound/provider-aws/actions/runs/3571953098
aws_chime_voice_connector_termination: https://github.com/upbound/provider-aws/actions/runs/3572160748
aws_chime_voice_connector_termination_credentials: https://github.com/upbound/provider-aws/actions/runs/3572210453
aws_cloud9_environment_ec2: https://github.com/upbound/provider-aws/actions/runs/3572222163
aws_cloud9_environment_membership: https://github.com/upbound/provider-aws/actions/runs/3572233650
aws_cloudcontrolapi_resource: https://github.com/upbound/provider-aws/actions/runs/3572248726

### CODE REVIEW

**Comment by mykolalosev** [comment]
/test-examples="examples/batch/schedulingpolicy.yaml"

**Comment by mykolalosev** [comment]
/test-examples="examples/budgets/budgetaction.yaml"

**Comment by mykolalosev** [comment]
/test-examples="examples/budgets/budgetaction.yaml"

**Comment by mykolalosev** [comment]
/test-examples="examples/chime/voiceconnector.yaml"

**Comment by mykolalosev** [comment]
/test-examples="examples/chime/voiceconnectorgroup.yaml"

**Comment by mykolalosev** [comment]
/test-examples="examples/chime/voiceconnectorlogging.yaml"

**Comment by mykolalosev** [comment]
/test-examples="examples/chime/voiceconnectororigination.yaml"

**Comment by mykolalosev** [comment]
/test-examples="examples/chime/voiceconnectorstreaming.yaml"

**Comment by mykolalosev** [comment]
/test-examples="examples/chime/voiceconnectortermination.yaml"

**Comment by mykolalosev** [comment]
/test-examples="examples/chime/voiceconnectorterminationcredentials.yaml"

**Comment by mykolalosev** [comment]
/test-examples="examples/cloud9/environmentec2.yaml"

**Comment by mykolalosev** [comment]
/test-examples="examples/cloud9/environmentmembership.yaml"

**Comment by mykolalosev** [comment]
/test-examples="examples/cloudcontrol/resource.yaml"

File: config/externalname.go [hunk 1]
```diff
@@ -1235,6 +1235,47 @@ var ExternalNameConfigs = map[string]config.ExternalName{
...
+	"aws_budgets_budget_action": config.IdentifierFromProvider,
+
+	// chime
+	//
+	// Configuration Recorder can be imported using the name
+	"aws_chime_voice_connector": config.NameAsIdentifier,
+	// Configuration Recorder can be imported using the name
+	"aws_chime_voice_connector_group": config.NameAsIdentifier,
+	// Chime Voice Connector Logging can be imported using the voice_connector_id
+	"aws_chime_voice_connector_logging": config.ParameterAsIdentifier("voice_connector_id"),
```
Comment (ulucinar) [inline config/externalname.go:1258]:
```suggestion
	"aws_chime_voice_connector_logging": config.IdentifierFromProvider,
```

**Comment by mykolalosev** [comment]
/test-examples="examples/chime/voiceconnector.yaml"

**Comment by mykolalosev** [comment]
/test-examples="examples/chime/voiceconnectorgroup.yaml"

**Comment by mykolalosev** [comment]
/test-examples="examples/chime/voiceconnectorlogging.yaml"

File: examples/budgets/budget.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+apiVersion: budgets.aws.upbound.io/v1beta1
+kind: Budget
+metadata:
+  annotations:
+    meta.upbound.io/example-id: budgets/v1beta1/budgetaction
```
Comment (ulucinar) [inline examples/budgets/budget.yaml:5]:
```suggestion
    meta.upbound.io/example-id: budgets/v1beta1/budget
```

File: examples/cloudcontrol/resource.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+kind: Resource
+metadata:
+  annotations:
+    meta.upbound.io/example-id: cloudcontrol/v1beta1/resource
+  labels:
+    testing.upbound.io/example-name: example
+  name: example
+spec:
+  forProvider:
+    desiredState: |-
```
Comment (ulucinar) [inline examples/cloudcontrol/resource.yaml:11]: nit: It should, in theory, be possible to remove the Terraform `jsonencode` operator here, this should also be okay.

File: examples/chime/voiceconnector.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+kind: VoiceConnector
+metadata:
+  annotations:
+    meta.upbound.io/example-id: chime/v1beta1/voiceconnector
+  labels:
+    testing.upbound.io/example-name: test
+  name: test
+spec:
+  forProvider:
+    awsRegion: us-west-1
```
Comment (ulucinar) [inline examples/chime/voiceconnector.yaml:11]: Looks like AWS does not allow `awsRegion` to be set to `us-west-1`:
```
1.66980215229754e+09    DEBUG   provider-aws    Cannot observe external resource        {"controller": "managed/chime.aws.upbound.io/v1beta1, kind=voiceconnector", "request": "/test", "uid": "9242f53c-03aa-4851-bb99-c9b8a3d89e17", "version": "2699", "external-name": "test", "error": "cannot run refresh: refresh failed: expected aws_region to be one of [us-east-1 us-west-2], got us-west-1: ", "errorVerbose": "refresh failed: expected aws_region to be one of [us-east-1 us-west-2], got us-west-1: \ncannot run refresh
```

According to Terraform docs, this field is optional. Could you please remove it from the example manifests?
```suggestion
```

File: examples/cloudcontrol/resource.yaml (hunk 3 above)
Comment (mykolalosev) [reply examples/cloudcontrol/resource.yaml:11]: I'he tried to use just string, like this:
        {
          "ClusterName": "example",
          "Tags": [
            {
              "Key": "CostCenter",
              "Value": "IT"
            }
          ]
        }
        
 but I got the error:
 cannot get terraform setup: failed to retrieve aws credentials from aws config: request canceled, context canceled

**Comment by mykolalosev** [comment]
/test-examples="examples/chime/voiceconnector.yaml"

**Comment by mykolalosev** [comment]
/test-examples="examples/chime/voiceconnectorgroup.yaml"

**Comment by mykolalosev** [comment]
/test-examples="examples/chime/voiceconnectorlogging.yaml"

**Review by ulucinar — APPROVED** [review]
Thank you @mykolalosev, lgtm.

**Pull request merged**